Can't resolve stringify-loader.js

I appears that version 0.28.6 introduced a bug related to the file watching mechanism.  When I alter a file I get the following error message:

```
ModuleNotFoundError: Module not found: Error: Can't resolve '/ncc-bug/node_modules/@vercel/ncc/dist/ncc/loaders/stringify-loader.js' in '/ncc-bug'
```

I've created a minimal reproduction of my setup here: https://github.com/kilahm/vercel-0.28.6-stringify-loader

This setup works (with the file watching) for `@vercel/ncc@0.28.5`, so [it seems like](https://github.com/vercel/ncc/compare/0.28.5...0.28.6) it was introduced by an update to either `ts-loader` or `webpack`.

--- a/scripts/build.js
+++ b/scripts/build.js
@@ -65,6 +65,12 @@ async function main() {
   );
   checkUnknownAssets('ts-loader', Object.keys(tsLoaderAssets).filter(asset => !asset.startsWith('lib/') && !asset.startsWith('typescript/lib')));
 
+  const { code: stringifyLoader, assets: stringifyLoaderAssets } = await ncc(
+    __dirname + "/../src/loaders/stringify-loader",
+    { filename: "stringify-loader.js", minify, cache, v8cache }
+  );
+  checkUnknownAssets('stringify-loader', Object.keys(stringifyLoader));
+
   const { code: sourcemapSupport, assets: sourcemapAssets } = await ncc(
     require.resolve("source-map-support/register"),
     { filename: "sourcemap-register.js", minify, cache, v8cache }
@@ -87,6 +93,7 @@ async function main() {
   writeFileSync(__dirname + "/../dist/ncc/loaders/relocate-loader.js.cache", relocateLoaderAssets["relocate-loader.js.cache"].source);
   writeFileSync(__dirname + "/../dist/ncc/loaders/shebang-loader.js.cache", shebangLoaderAssets["shebang-loader.js.cache"].source);
   writeFileSync(__dirname + "/../dist/ncc/loaders/ts-loader.js.cache", tsLoaderAssets["ts-loader.js.cache"].source);
+  writeFileSync(__dirname + "/../dist/ncc/loaders/stringify-loader.js.cache", stringifyLoaderAssets["stringify-loader.js.cache"].source);
 
   writeFileSync(__dirname + "/../dist/ncc/cli.js.cache.js", cliAssets["cli.js.cache.js"].source);
   writeFileSync(__dirname + "/../dist/ncc/index.js.cache.js", indexAssets["index.js.cache.js"].source);
@@ -94,6 +101,7 @@ async function main() {
   writeFileSync(__dirname + "/../dist/ncc/loaders/relocate-loader.js.cache.js", relocateLoaderAssets["relocate-loader.js.cache.js"].source);
   writeFileSync(__dirname + "/../dist/ncc/loaders/shebang-loader.js.cache.js", shebangLoaderAssets["shebang-loader.js.cache.js"].source);
   writeFileSync(__dirname + "/../dist/ncc/loaders/ts-loader.js.cache.js", tsLoaderAssets["ts-loader.js.cache.js"].source);
+  writeFileSync(__dirname + "/../dist/ncc/loaders/stringify-loader.js.cache.js", stringifyLoaderAssets["stringify-loader.js.cache.js"].source);
 
   writeFileSync(__dirname + "/../dist/ncc/cli.js", cli, { mode: 0o777 });
   writeFileSync(__dirname + "/../dist/ncc/index.js", index);
@@ -102,6 +110,7 @@ async function main() {
   writeFileSync(__dirname + "/../dist/ncc/loaders/relocate-loader.js", relocateLoader);
   writeFileSync(__dirname + "/../dist/ncc/loaders/shebang-loader.js", shebangLoader);
   writeFileSync(__dirname + "/../dist/ncc/loaders/ts-loader.js", tsLoader);
+  writeFileSync(__dirname + "/../dist/ncc/loaders/stringify-loader.js", stringifyLoader);
   writeFileSync(__dirname + "/../dist/ncc/loaders/uncacheable.js", readFileSync(__dirname + "/../src/loaders/uncacheable.js"));
   writeFileSync(__dirname + "/../dist/ncc/loaders/empty-loader.js", readFileSync(__dirname + "/../src/loaders/empty-loader.js"));
   writeFileSync(__dirname + "/../dist/ncc/loaders/notfound-loader.js", readFileSync(__dirname + "/../src/loaders/notfound-loader.js"));

--- a/src/loaders/stringify-loader.js
+++ b/src/loaders/stringify-loader.js
@@ -1,0 +1,1 @@
+module.exports = (source) => JSON.stringify(source);

